python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -104,9 +104,13 @@ class PsfexPsfDeterminerConfig(measAlg.BasePsfDeterminerConfig):
         dtype=bool,
         default=False,
     )
-
-    def setDefaults(self):
-        self.kernelSize = 41
+    kernelSize = pexConfig.Field(
+        doc=("Size of the postage stamp around each star that is extracted for fitting."
+             "Note: this reflects the oversampling setting of the psf, set by `samplingSize`;"
+             "e.g. `samplingSize=0.5` would require this value to be 2x what you expect."),
+        dtype=int,
+        default=81
```

Comment:
To be consistent with the other config fields, there should be a comma at the end. Otherwise, looks good.